packages/engine/lib/execution/src/package/simulation/output/creator.rs

```diff
@@ -16,20 +16,22 @@ pub struct OutputPackageCreators {
 }
 
 impl OutputPackageCreators {
-    pub fn from_config(_config: &PackageInitConfig) -> Result<Self> {
-        tracing::debug!("Initializing Output Package Creators");
-
-        let mut creators = HashMap::<_, Box<dyn OutputPackageCreator>>::with_capacity(2);
-        creators.insert(OutputPackageName::Analysis, Box::new(AnalysisCreator));
-        creators.insert(OutputPackageName::JsonState, Box::new(JsonStateCreator));
-        Ok(Self { creators })
+    pub fn initialize_for_experiment_run(_config: &PackageInitConfig) -> Result<&'static Self> {
+        static PACKAGE_CREATORS: SyncOnceCell<OutputPackageCreators> = SyncOnceCell::new();
+        PACKAGE_CREATORS.get_or_try_init(|| {
+            tracing::debug!("Outputializing Output Package Creators");
```

Comment:
These are kind of funny but I don't think we should keep them in I'm afraid :( They're just a bit confusing and I could see them actually being distracting while debugging.